tests/test_fsclient.py

```diff
@@ -18,20 +24,13 @@
 #         actual = f.read()
 #     assert actual == json_doc
 
-# todo:
-# build a runcontrol object as in regolith.  have it created globally in the
-# tests for  reuse in all the tests (look for DEFAULT_RC in regoith tests)
-# for now:
-# DEFAULT_RC = RunControl(
-#     _validators=DEFAULT_VALIDATORS,
-#     builddir="_build",
-#     mongodbpath=property(lambda self: os.path.join(self.builddir, "_dbpath")),
-#     user_config=os.path.expanduser("~/.config/regolith/user.json"),
-#     force=False,
-#     database=None
-# )
-DEFAULT_RC = {}
+
 rc = DEFAULT_RC
+with TempDirectory() as d:
```

Comment:
What is this block testing?  Do we need to write hte file then read it?  If it is testing load_rcfile, can we write it more like a test that asserts something (that load_rcfile returns reasonable things?)

Reply:
This block actually reads from pydr_rc.py and updates the runcontrol